test/extended/util/annotate/rules.go

```diff
@@ -68,7 +68,10 @@ var (
 			`\[sig-network-edge\]\[Feature:Idling\] Unidling should work with TCP \(while idling\)`,
 		},
 		// tests that may work, but we don't support them
-		"[Disabled:Unsupported]": {},
+		"[Disabled:Unsupported]": {
+			// https://bugzilla.redhat.com/show_bug.cgi?id=2019115: skip failing vSphere-specific storage tests
```

Comment:
Maybe add a note that tests with `In-tree Volumes [Driver: vsphere]` will still tests the volume plugin enough.

Reply:
Added a note